src/Files.App/MainWindow.xaml.cs

```diff
@@ -339,5 +342,15 @@ x.tabItem.NavigationParameter.NavigationParameter is PaneNavigationArguments pan
 				}
 			}
 		}
+
+		public double GetDpiScale()
```

Comment:
We already have `AppWindowDPI` in AppModel, is there any way to reuse it?

Reply:
Thanks, I didn't know.